docs/testnet/join.md

```diff
@@ -84,6 +84,13 @@ Edit the file located in `~/.evmosd/config/config.toml` and the `seeds` to the f
 seeds = "<node-id>@<ip>:<p2p port>"
 ```
 
+You can use following code to get seeds from repo and add it to your config
+```bash
+SEEDS=`curl -sL https://raw.githubusercontent.com/tharsis/testnets/main/olympus_mons/seeds.txt | awk '{print $1}' | paste -s -d, -`
+sed -i.bak -e "s/^seeds =.*/seeds = \"$SEEDS\"/" ~/.evmosd/config/config.toml
+```
+
+
```

Comment:
```suggestion
You can use the following code to get seeds from the repo and add it to your config:

```bash
SEEDS=`curl -sL https://raw.githubusercontent.com/tharsis/testnets/main/olympus_mons/seeds.txt | awk '{print $1}' | paste -s -d, -`
sed -i.bak -e "s/^seeds =.*/seeds = \"$SEEDS\"/" ~/.evmosd/config/config.toml
```

```

Reply:
Oh, sorry for my poor English, I was not very attentive after a sleepless night. Thank you for correcting my mistakes.